crates/core/build.rs

```diff
@@ -0,0 +1,36 @@
+use zksync_error_codegen::arguments::Backend;
+use zksync_error_codegen::arguments::GenerationArguments;
+
+fn main() {
+    // If we have modified anvil errors, forces rerunning the build script and
+    // regenerating the crate `zksync-error`.
+    println!("cargo:rerun-if-changed=../../etc/resources/anvil.json");
```

Comment:
Nit: AFAIR `cargo::KEY=VALUE` is the preferred syntax now (starting from Rust 1.77):
```suggestion
    println!("cargo::rerun-if-changed=../../etc/resources/anvil.json");
```